### PR TITLE
Add indices option

### PR DESCRIPTION
**Pull request by bgpat**
## WHY

X-Pack has enabled since Elasticsearch v6. X-Pack creates indices, which has `.` prefix for watches and monitoring. I expect to restore all indices except which has `.` prefix.

## WHAT

I added `--indices` option, and set default value to `*,-.*`.
(all indices except which has `.` prefix)

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -235,7 +237,7 @@ func createSnapshot() error {
 		Retry(options.MaxRetries, options.RetryInterval(), http.StatusGatewayTimeout).
 		Put(requestURL).
 		Send(&SnapshotSettings{
-			Indices:            "*",
+      Indices: options.Indices,
```
Comment (rerost) [inline main.go]: インデントが壊滅的に見える

Comment (bgpat) [reply main.go]: thx 🙏

**Comment by bgpat** [comment]
ping @rerost

File: main.go [hunk 2]
```diff
@@ -300,8 +302,7 @@ func restoreSnapshot() error {
 	_, _, errs = gorequest.New().
 		Retry(options.MaxRetries, options.RetryInterval(), http.StatusGatewayTimeout).
 		Post(requestURL).
-		Send(&SnapshotSettings{
-			Indices:            "*",
+		Send(&SnapshotSettings{Indices: options.Indices,
```
[the remark this answers is not in the document]
Comment (bgpat) [reply main.go]: https://godoc.org/github.com/parnurzeal/gorequest#SuperAgent.Send

**Pull request merged**